src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.v2.js

```diff
@@ -181,7 +192,8 @@ export function transformFormToVAOSAppointment(state) {
       desiredDate: `${data.preferredDate}T00:00:00+00:00`,
     },
     locationId: data.vaFacility,
+    reasonCode: getReasonCode(data),
     // removing this for now, it's preventing QA from testing, will re-introduce when the team figures out how we're handling the comment field
-    comment: getUserMessage(data),
+    // comment: getUserMessage(data),
```

Comment:
Since this is where the comment key:value pair is transformed for appointment request, we can add it for direct schedule appointment
```suggestion
    comment: data.reasonAdditionalInfo,
```

Reply:
Did not change since this change is in the 'transformFormToVAOSApppointment' (direct scheduling) function.